docker/rpi/rpi.mk

```
@@ -0,0 +1,10 @@
+BOARDS += rpi
+
+local-rpi: version
+	docker buildx bake --load --file=docker/rpi/bake.hcl --set rpi.tags=frigate:latest-rpi rpi
```

This file is named `rpi.hcl` instead of bake.hcl.  Same in other tarkets.
```suggestion
	docker buildx bake --load --file=docker/rpi/rpi.hcl --set rpi.tags=frigate:latest-rpi rpi
```